config-reloader/templates/kubernetes-postprocess.conf

```diff
@@ -49,14 +40,3 @@
   remove_keys kubernetes_namespace_container_name
 </filter>
 
-# Parse logs in the kube-system namespace using the kubernetes formatter.
-<filter kube.kube-system.**>
-  @type parser
-  reserve_data true
-  key_name log
-  emit_invalid_record_to_error false
-  <parse>
-    @type kubernetes
```

Comment:
@raja-gola I have looked at the change. I think it breaks number additional fields populated as part of the custom kubernetes parse plugin in kube-system ns. The parser removes the 'log' key and add some extra fields which will then be attached to the new key 'message'. I am planning to make a new release by end of this week. Would you mind trying again on the latest release ?

Reply:
@shukla2009 Can you provide more details about the issue? The details would be really helpful for finding a solution